src/Microsoft.Health.Client/Authentication/ICredentialProvider.cs

```diff
@@ -6,7 +6,7 @@
 using System.Threading;
 using System.Threading.Tasks;
 
-namespace Microsoft.Health.Client;
+namespace Microsoft.Health.Client.Authentication;
 
 public interface ICredentialProvider
```

Comment:
Perhaps outside the scope of these changes, but should `GetBearerToken` be `GetBearerTokenAsync`?